fbw-a380x/src/systems/instruments/src/MFD/pages/FMS/POSITION/MfdFmsPositionMonitor.tsx

```diff
@@ -0,0 +1,719 @@
+
+    if (updatePosition) {
+      irPosition.set(coordinateToString(coordinates, false));
+    }
+
+    if (fmPosition) {
+      irFmPositionDeviation.set(distanceTo(coordinates, fmPosition).toFixed(1));
+    } else {
+      irFmPositionDeviation.set(MfdFmsPositionMonitor.noIrsPositionDeviationAvailText);
+    }
+  }
+
+  private toggleSensorsVisibility() {
+    this.positionSensorsVisible.set(!this.positionSensorsVisible.get());
+    this.sensorsDisplayedByPilot = this.positionSensorsVisible.get();
+    if (!this.sensorsDisplayedByPilot) {
+      this.positionFrozen.set(false);
+    }
+  }
+
+  private togglePositionFrozen(): void {
+    this.positionFrozen.set(!this.positionFrozen.get());
+    // Automatically toggle position sensors visibility when freezing position
+    if (this.positionFrozen.get()) {
+      const sensorsVisible = this.positionSensorsVisible.get();
+      if (!sensorsVisible) {
+        this.positionSensorsVisible.set(true);
+        this.positionUpdateRequiredDueToFreeze = true;
+      }
+    } else {
+      // Only hide position sensors if the pilot didn't explicitly request them to be shown before the position was frozen
+      if (!this.sensorsDisplayedByPilot) {
+        this.positionSensorsVisible.set(false);
+      }
+    }
+  }
+
+  render(): VNode {
+    return (
+      <>
+        {super.render()}
+        {/* begin page content */}
+        <div class="mfd-page-container">
+          <div class="mfd-pos-top-row">
+            <div class="mfd-label-value-container" style={'margin-top:13px'}>
+              <span class={this.navPrimaryClass}>{this.navPrimaryText}</span>
+            </div>
+            <div class="rnp-container">
+              <div class="mfd-label-value-container">
+                <span class="mfd-label bigger mfd-spacing-right">RNP</span>
+                <InputField<number>
+                  dataEntryFormat={new RnpFormat()}
+                  value={this.fmsRnp}
+                  onModified={(v) => this.props.fmcService.master?.navigation.setPilotRnp(v)}
+                  enteredByPilot={this.rnpEnteredByPilot}
+                  canBeCleared={Subject.create(true)}
+                  containerStyle="width: 155px;"
+                  alignText="center"
+                  errorHandler={(e) => this.props.fmcService.master?.showFmsErrorMessage(e)}
+                  hEventConsumer={this.props.mfd.hEventConsumer}
+                  interactionMode={this.props.mfd.interactionMode}
+                  bigUnit={true}
+                />
+              </div>
+            </div>
+          </div>
+          <div class="mfd-pos-top-row">
+            <div class="mfd-label-value-container" style={{ visibility: this.accuracyVisibility }}>
+              <span class="mfd-label bigger mfd-spacing-right">ACCURACY</span>
+              <span class={this.accuracyClass}>{this.fmsAccuracy}</span>
+            </div>
+            <div class="mfd-label-value-container" style={'margin-right:95px'}>
```

Comment:
```suggestion
            <div class="mfd-label-value-container" style="margin-right:95px">
```
same here and in other instances